contracts/protocol/clients/voucher/BosonVoucher.sol

```diff
@@ -633,26 +650,22 @@ contract BosonVoucher is IBosonVoucher, BeaconClientBase, OwnableUpgradeable, ER
         address _to,
         uint256 _tokenId
     ) internal override {
-        // Update the buyer associated with the voucher in the protocol
-        // Only when transferring, not minting or burning
-        if (_from == owner()) {
```

Comment:
Why did you remove this line? 
The only issue related to this that I found was https://github.com/bosonprotocol/boson-protocol-contracts/issues/504 and their suggestion looks different

Reply:
Implementation is exactly what they suggest.

First they suggest changing `if (_from == owner())` to  `if (_from == owner() && _premintStatus.committable)`

So this is true if both are true, but as soon as `_premintStatus.committable == true`, it always means `_from == owner()`. So it completely sufficient to test only `_premintStatus.committable == true` as they suggest at very end. I just didn't do explicit check of equality `if(_premintStatus.committable == true)` since commitable is already bool.